content/en/docs/components/katib/katib-config.md

```diff
@@ -6,436 +6,393 @@ weight = 70
-   for the `File` metrics collector's container.
-
-   The default value is `IfNotPresent`
-
-1. `resources` - [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
-   for the `File` metrics collector's container. In the above example you
-   can check how to specify `limits` and `requests`. Currently, you can specify
-   only `memory`, `cpu` and `ephemeral-storage` resources.
-
-   The default values for the `requests` are:
-
-   - `memory = 10Mi`
-   - `cpu = 50m`
-   - `ephemeral-storage = 500Mi`
-
-   The default values for the `limits` are:
-
-   - `memory = 100Mi`
-   - `cpu = 500m`
-   - `ephemeral-storage = 5Gi`
-
-   You can run your metrics collector's container without requesting
-   the `cpu`, `memory`, or `ephemeral-storage` resource from the Kubernetes cluster.
-   For instance, you have to remove `ephemeral-storage` from the container resources to use the
-   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
-
-   To remove specific resources from the metrics collector's container set the
-   negative values in requests and limits in your Katib config as follows:
-
-   ```json
-   "requests": {
-     "cpu": "-1",
-     "memory": "-1",
-     "ephemeral-storage": "-1"
-   },
-   "limits": {
-     "cpu": "-1",
-     "memory": "-1",
-     "ephemeral-storage": "-1"
-   }
-   ```
-
-1. `waitAllProcesses` - a flag to define whether the metrics collector should
-   wait until all processes in the training container are finished before start
-   to collect metrics.
-
-   The default value is `true`
-
-## Suggestion settings
-
-These settings are related to Katib suggestions, where:
-
-- key: `suggestion`
-- value: corresponding JSON settings for each algorithm name
-
-If you want to use a new algorithm, you need to update the Katib config. For example,
-using a `random` algorithm with all settings looks as follows:
-
-```json
-suggestion: |-
-{
-  "random": {
-    "image": "docker.io/kubeflowkatib/suggestion-hyperopt",
-    "imagePullPolicy": "Always",
-    "resources": {
-      "requests": {
-        "memory": "100Mi",
-        "cpu": "100m",
-        "ephemeral-storage": "100Mi"
-      },
-      "limits": {
-        "memory": "500Mi",
-        "cpu": "500m",
-        "ephemeral-storage": "3Gi"
-      }
-    },
-    "serviceAccountName": "random-sa"
-  },
-  ...
-}
+It has settings for the following Katib components:
+
+1. Katib certificate generator: `certGenerator`
+
+1. Katib controller: `controller`
+
+### Katib Certificate Generator Parameters
+
+The following parameters set in `.init.certGenerator` configure the Katib certificate generator:
+
+- `enable` - whether to enable Katib certificate generator.
+
+  The default value is `true`
```

Comment:
```suggestion
  The default value is `false`
```

This parameter's default is `false`: https://github.com/kubeflow/katib/blob/19268062f1b187dde48114628e527a2a35b01d64/pkg/apis/config/v1beta1/types.go#L88

Reply:
Nice catch, Golang set `false` as default value for bool.